src/Command/PHPUnitCommand.php

```diff
@@ -108,7 +115,19 @@ private function resolveBinary(InputInterface $input)
             return '';
         }
 
-        return 'phpdbg -d memory_limit=-1 -qrr ';
+        // Depending on the coverage driver, selected return different values.
+        switch ($this->resolveCoverageDriver($input, $output)) {
+            case 'pcov':
+                return 'php -dxdebug.mode=off -dpcov.enabled=1 -dpcov.directory=. '; // Eable pcov, disable xdebug, just in case.
```

Comment:
Hi @stronk7,
small typo here in the comment: `Eable` => `Enable`.

HTH,
Matteo

Reply:
Thanks @scara , amended!